### PR TITLE
1.14/1867 as an admin when i download my registrations i can see that the main participants age is flagged or first so that i can sort pro

### PR DESCRIPTION
**Pull request by neonleann**
https://trello.com/c/4bx8IwjL/1867-as-an-admin-when-i-download-my-registrations-i-can-see-that-the-main-participants-age-is-flagged-or-first-so-that-i-can-sort-pro

### CODE REVIEW

File: tests/Unit/Controllers/Store/CentreControllerTest.php [hunk 1]
```diff
@@ -208,7 +218,7 @@ public function testItCanDownloadASocialPrescriptionsRegistrationsSpreadsheet()
             "Eligible From",
             "Due Date",
         ];
-
+\Log::info($headers);
```
Comment (lauragurney) [inline tests/Unit/Controllers/Store/CentreControllerTest.php:221]: Did you want to keep this here or was it an accident?

Comment (neonleann) [reply tests/Unit/Controllers/Store/CentreControllerTest.php]: 😮 Well spotted!

**Review by lauragurney — APPROVED** [review]
Lully job 🎩

**Review by charlesstrange2 — APPROVED** [review]
lovely ✅

**Pull request merged**